src/components/MenuItemList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import _ from 'underscore';
+import PropTypes from 'prop-types';
+import MenuItem from './MenuItem';
+import MenuItemPropTypes from './MenuItemPropTypes';
+
+const propTypes = {
+    /** An array of props that are pass to individual MenuItem components */
+    menuItems: PropTypes.arrayOf(PropTypes.shape(MenuItemPropTypes)),
+};
+const defaultProps = {
+    menuItems: [],
+};
+
+const MenuItemList = ({menuItems}) => (
+    <>
+        {_.map(menuItems, menuItemProps => (
+            <MenuItem
+                key={_.uniqueId(menuItemProps.title)}
```

Comment:
Yeah my main issue is not that this is causing some immediate problem but it's not a great example to leave behind for others to copy and we should try to use keys correctly.

Here's an alternative: 

Give the `menuItemProps` a `key` field and explicitly ensure the keys are unique.

e.g. if we are trying to prevent duplicate keys for something that might have the same title then it's up to whoever is using this to give it a key like `${policyID}_${policyName}`.